lib/metric_fu/metrics/flog/generator.rb

```diff
@@ -1,5 +1,6 @@
 require "pathname"
 require "optparse"
+require "path_expander"
```

Comment:
since this gem is a dependency of flog, it would be better to require it through flog, or to add it as a metric_fu dep (not preferred)

looks like `require 'flog_cli'` should be sufficient